networking_cisco/plugins/cisco/cfg_agent/device_drivers/asr1k/aci_asr1k_routing_driver.py

```diff
@@ -248,24 +293,49 @@ def cleanup_invalid_cfg(self, hd, routers):
         cfg_syncer = syncer.ConfigSyncer(routers, self, hd)
         cfg_syncer.delete_invalid_cfg()
 
+    def _create_vrf(self, ri):
+        # just get the VRF -- creation is done
+        # by the service handler
+        self._get_vrf_name(ri)
+
+    def _do_create_vrf(self, vrf_name):
+        vrf_str = snippets.VRF_CONFIG % vrf_name
```

Comment:
I noticed "vrf definition %s" string already exists in csr_snippets.CREATE_VRF command set.....In that case do we still need to use snippets.VRF_CONFIG then?